### ISSUE TITLE
Only first line of commit message shown in Commit tab 

### ISSUE THREAD

**Issue by minusf** [ForkIssues/Tracker#234]
It would be nice to see the whole commit message under the "Commit" tab, not just the first line.
The "Commit message" being called "Subject" is also misleading...

**Comment by DanPristupov** [comment]
It's there.
![2018-03-01 at 15 59](https://user-images.githubusercontent.com/618115/36851518-ae4f0b04-1d69-11e8-9531-974f21abdc30.png)

**Comment by minusf** [comment]
very confusing. some tools show this comment as a long 1 line comment...

```
$ git log
commit ee17d55b362d5cb34c08cd64493f0dd9649be084
Author: Frantisek Holop <fholop@ripe.net>
Date:   Thu Mar 1 15:07:53 2018 +0100

    - upgrade some requirements,
    - add django-extensions for dev,
    - make deploy_check for checking out of date packages and template syntax
```
<img width="1098" alt="screen shot 2018-03-02 at 15 51 55" src="https://user-images.githubusercontent.com/3632883/36904854-1e39bebe-1e32-11e8-8921-899567df2f2a.png">

**Comment by minusf** [comment]
btw my comment about "Subject" still holds...

**Comment by DanPristupov** [comment]
@minusf it is causes by incorrect commit format. Commit header and description must be separated by **two** newline symbols: `\n\n`.
Example:
`git commit -a -m $'Commit title\n\nRest of commit message...'`
Or
`git commit -m "Commit title" -m "Rest of commit message..."`

Regarding the label.
> btw my comment about "Subject" still holds...

Yes, I agree. Going to rename `Subject:` to `Message:` in the next update.

**Comment by minusf** [comment]
What a mess.

```
$ git help commit
...
       -m <msg>, --message=<msg>
           Use the given <msg> as the commit message. If multiple -m options are given,
           their values are concatenated as separate paragraphs.
...
DISCUSSION
       Though not required, it's a good idea to begin the commit message with
       a single short (less than 50 character) line summarizing the change,
       followed by a blank line and then a more thorough description. The text
       up to the first blank line in a commit message is treated as the commit title,
       and that title is used throughout Git. For example, git-format-patch(1)
       turns a commit into email, and it uses the title on the Subject line and
       the rest of the commit in the body.
...
```

So it's not "required" to have this format.  I think that if there is no blank line after
the first line, the message shouldn't be considered following this unofficial "format".

**Comment by FinesseRus** [comment]
I join this issue. I recently moved to Fork from SourceTree, which uses a single multiline field for commit message, so all my commits have multiline subject and Fork shows only the first lines of the messages.

**Comment by minusf** [comment]
fix for this is not in 1.0.65, correct?

**Comment by DanPristupov** [comment]
I improved that in 1.0.68.
